### PR TITLE
[DO NOT MERGE] Add new swagger for anti malware

### PR DESCRIPTION
**Pull request by eswatad**

### CODE REVIEW

**Review by ruowan — APPROVED** [review]

File: specification/security/resource-manager/Microsoft.Security/preview/2021-08-01-preview/antiMalwareSettings.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AntiMalwareProperties": {
+      "type": "object",
+      "properties": {
+        "isEnabled": {
```
Comment (j5lim) [inline specification/security/resource-manager/Microsoft.Security/preview/2021-08-01-preview/antiMalwareSettings.json:121]: >"isEnabled": { [](http://example.com/codeflow?start=8&length=14)

Please avoid using a boolean type. Instead model it as a string enum. e.g. "status": ["enabled", "disabled"]. It is more flexible and additional values can be added later without a breaking change.

[https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review)

File: specification/security/resource-manager/Microsoft.Security/preview/2021-08-01-preview/antiMalwareSettings.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{resourceId}/providers/Microsoft.Security/antiMalwareSettings/{settingName}": {
```
Comment (j5lim) [inline specification/security/resource-manager/Microsoft.Security/preview/2021-08-01-preview/antiMalwareSettings.json:37]: >/{resourceId}/providers/Microsoft.Security/antiMalwareSettings/{settingName} [](http://example.com/codeflow?start=5&length=76)

Please add list operation as well.

[https://armwiki.azurewebsites.net/api_contracts/guidelines/policy.html#plcy008-support-a-collection-get-and-individual-get-for-child-resource-types-even-singletons](https://armwiki.azurewebsites.net/api_contracts/guidelines/policy.html#plcy008-support-a-collection-get-and-individual-get-for-child-resource-types-even-singletons)

File: specification/security/resource-manager/Microsoft.Security/preview/2021-08-01-preview/antiMalwareSettings.json [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
```
Comment (j5lim) [inline specification/security/resource-manager/Microsoft.Security/preview/2021-08-01-preview/antiMalwareSettings.json:75]: >  "put": { [](http://example.com/codeflow?start=4&length=10)

Is delete not allowed?

Comment (eswatad) [reply specification/security/resource-manager/Microsoft.Security/preview/2021-08-01-preview/antiMalwareSettings.json:75]: added

**Review by j5lim — APPROVED** [review]
Approved from ARM side.

**Pull request closed without merging**